playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+| STAGING_KEY   | The GCP KMS online key for the staging delegation.       |   `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/staging`      |
+| REVOCATION_KEY   | The GCP KMS online key for the revocation delegation, containing any targets that were explicitly revoked.       |    `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/revocation`     |
+
+## Configuration
+
+Each root signing event occurs inside a folder `ceremony/YYYY-MM-DD/` named by the date the signing event started. This folder contains:
+* A `keys/` subfolder containing subdirectories named after Yubikey serial numbers, and containins public key PEMs, key certificates, and device certificates attesting to the hardware key. See [PIV attestation](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html). 
+* A `repository/` subfolder containing the finalized TUF repository metadata.
+* A `staging/` subfolder present during root signing events with staged metadata before publishing.
+
+### Targets and Delegation configuration
+
+For top-level targets and delegation files, a YAML configuration file named `role-metadata.yml` specifies the target files for the role and their [custom metadata](https://theupdateframework.github.io/specification/latest/#custom). We use [Sigstore custom metadata](https://github.com/sigstore/sigstore/blob/ec8f2d403e07a392ea363560d21c31aaee57ba0f/pkg/tuf/client.go#L95) to allow Sigstore clients to identity usage of the targets. Target files are customarily located in the [targets/] subfolder. For example, this defines the target file located at `targets/fulcio_v1.crt.pem` with custom metadata to indicate Fulcio root usage.
+
+```yaml
+targets/fulcio_v1.crt.pem:
+  sigstore:
+    usage: Fulcio
+    status: Active
+```
+
+### Key configuration
+
+There are two types of keys that are used during root signing. 
+
+First, we use hardware Yubikeys for root and target signers. There are 5 [root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders) each containing one key. A [default threshold](https://github.com/sigstore/root-signing/blob/e3f1fe5e487984f525afc81ac77fa5ce39737d0f/cmd/tuf/app/init.go#L24) of 3 are required to sign the root and targets. This is configurable through a `threshold` flag on [initialization](#step-2-initializing-a-root-and-targets). Configuring the hardware keyholders is done through management of the `keys/` subfolder, see [Key Management](#step-1-root-key-updates) during the ceremony.
+
+Second, online keys on GCP are used for snapshot, timestamp, and delegation roles. These are defined by a [go-cloud](https://gocloud.dev) style URI to refer to the specific provider like `gcpkms://`. See cosign [KMS integrations](https://github.com/sigstore/cosign/blob/main/KMS.md) for details. These are configured through environment variables in the `scripts/` which propogate to command line flags to the binary.
+
```

Comment:
Do we need to record key rotations? There's an open question if we need to record all signing events. If trust is formed from the initial root, and that is recorded, that may be sufficient.

Reply:
I meant recording over GVC, like what was done for the first ceremony. Do we need that step?